app/services/import_references_from_csv.rb

```diff
@@ -0,0 +1,59 @@
+require 'csv'
+
+class ImportReferencesFromCsv
```

Comment:
Can you put this into a `SupportInterface` namespace?

Reply:
Update: Our services aren't currently namespaced. Leaving this as-is for now.